common/src/main/java/org/apache/falcon/util/StartupProperties.java

```diff
@@ -37,6 +39,9 @@
     public static final String SAFEMODE_PROPERTY = "falcon.safeMode";
     private static final String SAFEMODE_FILE = ".safemode";
     private static final String CONFIGSTORE_PROPERTY = "config.store.uri";
+    private static final String CREDENTIAL_PROVIDER_PROPERTY = "hadoop.security.credential.provider.path";
+    private static final String ALIAS_PROPERTY_PREFIX = "hadoop.security.alias.";
```

Comment:
Instead of saying hadoop.security.alias we should just say "credential.provider.alias.for" (which allows us to decouple from implementation specifics in future in terms of names).   The for at end makes it easy to understand the purpose